src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -117,6 +118,15 @@ public function getFilters()
         ];
     }
 
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('deprecate_template',
```

Comment:
Could you move this kind of stuff to twig itself?

Reply:
Nope: https://github.com/twigphp/Twig/issues/2599

Reply:
What about the https://github.com/sonata-project/SonataCoreBundle?

We could use template deprecations in other bundles too.

Reply:
@core23 that was just decided [here](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/772#discussion_r155790760).